easy_phi/app.py

```diff
@@ -375,9 +378,13 @@ def get_current_user(self):
         return auth.user_by_token(
             self.get_cookie(options.session_cookie_name))
 
+    @tornado.web.authenticated
```

Comment:
instead of putting decorator on get, put, post etc it is much simpler to wrap .prepare in base class. Just a code layout optimization